e2e/app/component/review-criteria-sidebar.ts

```diff
@@ -32,6 +33,9 @@ export default class ReviewCriteriaSidebar extends BaseHelpSidebar {
     await super.waitUntilVisible();
     const title = await this.getTitle();
     await this.waitUntilSectionVisible(SectionSelectors.SelectionList);
+    await Promise.race([this.findButton(LinkText.SaveCriteria), this.findButton(LinkText.AddThis)]);
+    await Promise.race([this.findButton('BACK'), this.findButton('Back')]);
+    await waitWhileLoading(this.page);
```

Comment:
I suspect `cohort-create.spec` test failed because the sidebar wasn't ready or was still `loading` when `waitForLoad` has finished.